### PR TITLE
refactor(storage): remove protobuf and old rocksdb storage

### PR DESCRIPTION
**Pull request by jansegre**
Acceptance criteria: have no traces of protobuf left (which was only used on the deprecated rocksdb storage)

### CODE REVIEW

**Review by pedroferreira1 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -88,32 +88,6 @@ isort:
...
-%_pb2.pyi %_pb2.py: %.proto $(proto_srcs)
-	$(protoc) -I. --python_out=. --mypy_out=. $<
-
-.PHONY: protos
-protos: $(proto_outputs)
-
-# cleaning:
-
-.PHONY: clean-protos
-clean-protos:
```
Comment (msbrogli) [inline Makefile:114]: You also have to remove `clean-protos` from `clean`.

File: hathor/transaction/storage/rocksdb_storage.py [hunk 2]
```diff
@@ -108,20 +108,6 @@ def _import_from_other(self, other_storage: TransactionStorage) -> None:
...
-        from rocksdb.errors import RocksIOError
-        if self.is_empty():
-            # try to load the old rocksdb storage
-            from hathor.transaction.storage.old_rocksdb_storage import TransactionOldRocksDBStorage
-            try:
-                old_storage = TransactionOldRocksDBStorage(self._path)
-            except RocksIOError:
-                # old storage does not exist, no need to continue
-                return
-            self._import_from_other(old_storage)
```
Comment (msbrogli) [inline hathor/transaction/storage/rocksdb_storage.py:121]: Should we remove the method `_import_from_other(...)` as well?

**Review by msbrogli — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (jansegre) [reply Makefile:114]: Oh, you're right!

File: hathor/transaction/storage/rocksdb_storage.py (hunk 2 above)
Comment (jansegre) [reply hathor/transaction/storage/rocksdb_storage.py:121]: Yes!

File: Makefile (hunk 1 above)
Comment (jansegre) [reply Makefile:114]: Done.

File: hathor/transaction/storage/rocksdb_storage.py (hunk 2 above)
Comment (jansegre) [reply hathor/transaction/storage/rocksdb_storage.py:121]: Done.

**Pull request merged**